ui/src/views/DeferredQueue.vue

```diff
@@ -49,20 +65,39 @@
               :icon="Email20"
               class="mg-left"
               @click="setResendQueueAll"
-              :disabled="loading.listDeferredQueue || loading.setDeleteQueue"
+              :disabled="
+                loading.listDeferredQueue ||
+                loading.setDeleteQueue ||
+                loading.getQueueSettings
+              "
               >{{ $t("queue.resend_all") }}
             </NsButton>
             <NsButton
               kind="danger"
               class="mg-left"
               :icon="TrashCan20"
               @click="toggleDeleteQueueAll"
-              :disabled="loading.listDeferredQueue || loading.setDeleteQueue"
+              :disabled="
+                loading.listDeferredQueue ||
+                loading.setDeleteQueue ||
+                loading.getQueueSettings
+              "
               >{{ $t("queue.delete_all") }}
             </NsButton>
           </template>
         </cv-column>
       </cv-row>
+      <cv-row>
+        <cv-column>
+          <div class="mg-top-sm mg-bottom-lg">
+            <NsSvg :svg="InformationFilled16" class="icon ns-info" />
+            <span>
+              {{ $t("settings_queue.maximal_queue_lifetime") }}
+            </span>
+            {{ maximal_queue_lifetime + " " + $t("settings_queue.hours") }}
```

Comment:
String concatenation inside i18n strings is not a good practice, let's use a parameter (and pluralization with `$tc` if possible)

Reply:
I updated like davidep asked, it should be good, could you confirm